src/Microsoft.Identity.Web/AppServicesAuth/AppServicesAuthenticationTokenAcquisition.cs

```diff
@@ -15,9 +16,15 @@ namespace Microsoft.Identity.Web
     /// <summary>
     /// Implementation of ITokenAcquisition for App Services authentication (EasyAuth).
     /// </summary>
-    public class AppServicesAuthenticationTokenAcquisition : ITokenAcquisition
+    public class AppServicesAuthenticationTokenAcquisition : ITokenAcquisition, IDisposable
```

Comment:
Although not a breaking change, apps consuming this type will incur a compiler warning that they should dispose of this object properly.

Reply:
Updated to remove IDisposable now that initialization is synchronous.